scrapers/extrapackage.yml

```diff
@@ -0,0 +1,76 @@
+name: extrapackage
+sceneByURL:
+  - action: scrapeXPath
+    scraper: sceneScraper
+    url:
+      - extrapackage.com
+movieByURL:
+  - action: scrapeXPath
+    scraper: movieScraper
+    url:
+      - extrapackage.com
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="boxGradientTop gradient section"]//p[contains(.,"Information")]
+        postProcess:
+          - replace:
+              - regex: 'Information - \s*'
+                with: ""
+      Date:
+        selector: //div[@class="wrpHighlightUpdated"]
+        postProcess:
+          - parseDate: 2006-01-02
+      Performers:
+        Name: //li[contains(.,"Pornstars:")]/a
+      Tags:
+        Name: //li[contains(.,"Categories :")]/a
+      Details:
+        selector: //div[@class="video-info mt-3"]/p
+      Movies:
+        Name: //div[@class="gblViewInfo"]//div[contains(.,"DVD :")]/a
+      URL:
+        selector: //div[@class="gblViewInfo"]//div[contains(.,"DVD :")]/a/@href
+        postProcess:
+          - replace:
+            - regex: ^
+              with: "http://www.extrapackage.com"
```

Comment:
this needs two spaces in every line to align with the `Name` as it is the URL of  the movie and not the scene